src/core/lender/assessor/base.sol

```diff
@@ -89,6 +91,7 @@ contract BaseAssessor is Math, Auth {
     }
 
     function calcTokenPrice(address tranche) public returns (uint) {
+        require(tranche  == junior || tranche == senior);
```

Comment:
I think the assessor should only give you numbers for senior and junior.

Theoretically, somebody could implement the interface in a contract and we would return a number

Reply:
yes